Source/Details/ASCollectionViewFlowLayoutCalculator.m

```diff
@@ -0,0 +1,35 @@
+//
+//  ASCollectionViewFlowLayoutCalculator.m
+//  AsyncDisplayKit
+//
+//  Created by Huy Nguyen on 28/2/17.
+//  Copyright © 2017 Facebook. All rights reserved.
+//
+
+#import <AsyncDisplayKit/ASCollectionViewFlowLayoutCalculator.h>
+#import <AsyncDisplayKit/ASCellNode.h>
+#import <AsyncDisplayKit/ASCollectionElement.h>
+#import <AsyncDisplayKit/ASElementMap.h>
+#import <AsyncDisplayKit/ASStackLayoutSpec.h>
+
+@implementation ASCollectionViewFlowLayoutCalculator
+
+- (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)sizeRange forElementMap:(ASElementMap *)map
+{
+  NSMutableArray<ASCellNode *> *children = [NSMutableArray array];
+  [map enumerateUsingBlock:^(NSIndexPath * _Nonnull indexPath, ASCollectionElement * _Nonnull element, BOOL * _Nonnull stop) {
+    [children addObject:element.node];
+  }];
+  
+  ASStackLayoutSpec *stackSpec = [ASStackLayoutSpec stackLayoutSpecWithDirection:ASStackLayoutDirectionHorizontal
+                                                                         spacing:0
+                                                                  justifyContent:ASStackLayoutJustifyContentStart
+                                                                      alignItems:ASStackLayoutAlignItemsStart
+                                                                        flexWrap:ASStackLayoutFlexWrapWrap
+                                                                    alignContent:ASStackLayoutAlignContentStart
+                                                                        children:children];
+  stackSpec.multithreaded = YES;
```

Comment:
different API, but we should decide if we want to use the word multithreaded here.  I think .concurrent is more appropriate; multithreaded harkens back to NSThread, whereas concurrent queues are the GCD terminology.